tests/Feature/Api/PublicScheduleTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Models\Lecturing;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class PublicScheduleTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function can_get_schedules_within_specified_date_range() {
```

Comment:
https://www.php-fig.org/psr/psr-12/#44-methods-and-functions
> The opening brace MUST go on its own line, and the closing brace MUST go on the next line following the body.
```suggestion
    public function can_get_schedules_within_specified_date_range()
    {
```